### PR TITLE
Update about non-analytic queries

### PR DESCRIPTION
**Pull request by sreekanth-cb**
update about the non-analytic queries like prefix, regex and wildcard.

### CODE REVIEW

**Comment by sreekanth-cb** [comment]
@amarantha-k , @abhinavdangeti please review.

@amarantha-k , whats the plan/process for back porting or making these doc changes available to other platforms as well?

I could see that a few 6.5 doc updates from dev not reflecting on 6.6 documentation. How can we fix this? :)

File: modules/fts/pages/fts-query-types.adoc [hunk 1]
```diff
@@ -346,6 +349,8 @@ More detailed information is provided in xref:query-string-queries.adoc[Query St
 _Term_ and _Phrase_ queries support no analysis on their inputs.
 This means that only exact matches are returned.
 
+Please note that xref:fts-query-types.adoc#prefix-query[Prefix], xref:fts-query-types.adoc#regexp-query[Regex] and xref:fts-query-types.adoc#wildcard-query[Wildcard] queries are also non-analytic queries.
```
Comment (abhinavdangeti) [inline modules/fts/pages/fts-query-types.adoc]: You've already pointed out earlier that prefix and wildcard queries are non-analytic. Why again here in the same page?

File: modules/fts/pages/fts-query-types.adoc [hunk 2]
```diff
@@ -379,6 +384,7 @@ It performs an exact term-match for all the phrase-constituents, without using a
 
 A phrase query is also demonstrated by means of the Java SDK, in xref:2.7@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
 
+
```
Comment (abhinavdangeti) [inline modules/fts/pages/fts-query-types.adoc]: Unnecessary new line. Also need to mention that fuzzy queries are non-analytic too.

File: modules/fts/pages/fts-query-types.adoc (hunk 1 above)
Comment (sreekanth-cb) [reply modules/fts/pages/fts-query-types.adoc]: People reading about non-analytic queries should get a list of such ones FTS supports. Hence felt that it is important to have a single place where it list all of them

File: modules/fts/pages/fts-query-types.adoc (hunk 2 above)
Comment (sreekanth-cb) [reply modules/fts/pages/fts-query-types.adoc]: done!

**Comment by amarantha-k** [comment]
@sreekanth-cb Sorry to hear that some of your updates are missing in 6.6. 
As a process, writers submit separate PRs to backport changes to other versions. If you need a change to be backported, we request that you add a note on the PR. 
We do take care of merging forward changes from the current version(say 6.6) to the next GA version (7.0) regularly.

**Review by amarantha-k — APPROVED** [review]

**Review by abhinavdangeti — APPROVED** [review]

**Pull request merged**